src/themes/components/button.ts

```diff
@@ -9,7 +9,7 @@ const baseStyle: SystemStyleObject = {
 const buttonSizes = {
   md: {
     height: 'md',
-    textStyle: 'button-sm',
```

Comment:
I will check with the UX-Designers if we still need the button-small token for future designs. Otherwise, I will remove it (So far, I have checked and we don't use this typography).